src/textToArt.ts

```diff
@@ -0,0 +1,36 @@
+import AinftBase from './ainftBase';
+import {
+  HttpMethod,
+  TextToArtResponse,
+  TextToArtParams,
+  TextToArtTxHash,
+} from './types';
+
+export default class PersonaModels extends AinftBase {
+  getTextToArtResults(
```

Comment:
Could you add a description of these functions as jsDoc?

Reply:
Ok. I added jsDoc